providers/from-env/README.md

```diff
@@ -0,0 +1,113 @@
+# Environment Variable JSON Flag Provider
+
+This repository contains a very simple environment variable based feature flag provider.  
+This provider uses a very simple JSON evaluation for matching a flag `Variant` to a provided `EvaluationContext`. Each flag `Variant` contains a slice of `Conditions`, if all `Conditions` match then the flags value is returned. Each `Variant` is evaluated starting at index 0, therefore the first matching `Variant` is returned. Each variant also has a `TargetingKey`, when set it must match the `TargetingKey` provided in the `EvaluationContext` for the `Variant` to be returned.  
+
+
+## Flag Configuration Structure. 
+
+Flag configurations are stored as JSON strings, with one configuration per flag key. An example configuration is described below.  
+```json
+{
+    "defaultVariant":"not-yellow",
+    "variants": [
+        {
+			"name": "yellow-with-key",
+            "targetingKey":"user",
+            "criteria": [
+                {
+                    "color":"yellow"
+                }
+            ],
+            "value":true
+        },
+		{
+			"name": "yellow",
+            "targetingKey":"",
+            "criteria": [
+                {
+                    "color":"yellow"
+                }
+            ],
+            "value":true
+        },
+        {
+			"name": "not-yellow",
```

Comment:
minor but there's weird spacing here, can this be fixed?

Reply:
fixed :)